Description: Fixes programs autostart twice when running in an Openbox session
Author: Eugenio Paolantonio <me@medesimo.eu>

--- a/data/autostart/openbox-autostart.in
+++ b/data/autostart/openbox-autostart.in
@@ -27,8 +27,3 @@
 elif test -f $AUTOSTART.sh; then
     sh $AUTOSTART.sh
 fi
-
-# Run the XDG autostart stuff.  These are found in /etc/xdg/autostart and
-# in $HOME/.config/autostart.  This requires PyXDG to be installed.
-# See openbox-xdg-autostart --help for more details.
-@libexecdir@/openbox-xdg-autostart "$@"
